influxdb/2.0/Dockerfile

```diff
@@ -60,3 +60,4 @@ EXPOSE 8086
 ENV INFLUX_CONFIGS_PATH /etc/influxdb2/influx-configs
 ENV INFLUXD_INIT_PORT 9999
 ENV INFLUXD_INIT_PING_ATTEMPTS 600
+ENV DOCKER_INFLUXDB_INIT_CLI_CONFIG_NAME default
```

Comment:
Why does this need to be set here for the `CLI_CONFIG_NAME`, whereas other `DOCKER_INFLUXDB_INIT_...` variables aren't?

Reply:
None of the other `DOCKER_INFLUXDB_INIT_` variables have defaults. It is odd to see all the different prefixes here, but I like having all the defaults in one place